charts/prefect-worker/values.yaml

```diff
@@ -59,7 +59,7 @@ worker:
     ## prefect tag is pinned to the latest available image tag at packaging time.  Update the value here to
     ## override pinned tag
     # -- prefect image tag (immutable tags are recommended)
-    prefectTag: 2-python3.11-kubernetes
+    prefectTag: 3-latest-kubernetes
```

Comment:
dumb q, do we need to specify a python version?

Reply:
hm maybe we should, otherwise it defaults to the latest python version (3.12.5)
```
ENV PYTHON_VERSION=3.12.5
```

Reply:
reverted to 3.11